server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessageFactoryTest.java

```diff
@@ -58,16 +63,31 @@
     private MessageFactory messageFactory;
     private MessagePreviewGenerator messagePreview ;
     private HtmlTextExtractor htmlTextExtractor;
-    
+
+    @Rule
+    public TikaContainer tika = new TikaContainer();
+
     @Before
-    public void setUp() {
-        htmlTextExtractor = new MailboxBasedHtmlTextExtractor(new TikaTextExtractor());
+    public void setUp() throws Exception {
+        tika.start();
+        TikaTextExtractor textExtractor = new TikaTextExtractor(new TikaHttpClientImpl(TikaConfiguration.builder()
+                .host(tika.getIp())
+                .port(tika.getPort())
+                .timeoutInMillis(tika.getTimeoutInMillis())
+                .build()));
+        htmlTextExtractor = new MailboxBasedHtmlTextExtractor(textExtractor);
```

Comment:
The goal of this task is to extract, not change all the tests behavior.
I'm afraid of too much responsibility on this issue, then it will take a long time to be approved/merged.